doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,93 @@
+[id='proc_kogito-deploying-on-ocp-console']
+
+= Deploying {PRODUCT} services on {OPENSHIFT} using the OpenShift web console
+
+After you create your {PRODUCT} services as part of a business application, you can use the {OPENSHIFT} web console to deploy your services. As an aid to deploy your services, you can also use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can pull the project, build it (including native compilation, if applicable), and deploy the resulting services.
```

Comment:
same than before about operator pulling the git repo..

Reply:
Commented above.